libraries/abstractions/pkcs11/test/MBT_C_FindObjectsInit.c

```diff
@@ -34,18 +34,20 @@
 
 #include "iot_test_pkcs11_globals.h"
 
+char* label = "Device Priv TLS Key";
+int labelLen = 19;
```

Comment:
Rather than hard-coded length, why not just call strlen after the declaration of the template, but before passing it into the API?

Reply:
Changed in 8ccfd49.